source/manual/data-gov-uk-deployment.html.md

```diff
@@ -75,6 +75,14 @@ Builds can take up to 10 minutes. You can view progress in GitHub Actions under
 
 Deployments typically take 5 minutes. You can view progress in Argo CD.
 
+### How to check that your release has been successfully deployed
+
+In order to validate that your release has been deployed you will need to check the relevant pod in the relevant environment for Argo CD.
+
+For changes to `CKAN` you will need to look under the `ckan` application and for the `Find` app you will need to look under the `datagovuk` application.
+
+If the Argo app health status is not green then the deployment has not been successful and will need investigating before you continue testing the web application as you will not be testing your release but the previous release as that will still be running.
```

Comment:
Is there's a more explicit way in Argo to check what version (commit SHA) is currently running in the container?

Reply:
what do you mean, by an inspection of the source page? I don't think thats necessary, I'll add another step to check that the cluster is synced with the HEAD of the main branch commit though as it might not have synced with the PR merge

Reply:
What I'd want to know is how can I check *in Argo* what version is deployed (currently running).  If it's the SYNC STATUS then would be good mention it.

The way I'd check it outside of Argo (which maybe a bit cowboy), would be to ssh into the machine and `cat` a file I expect to be changed to see if the change is there.